### DISCUSSION TITLE
I can't access web ui

### DISCUSSION THREAD

**Discussion by playaz44** [postalserver/postal#2984] [Help with using Postal] [answered]
Hello, could someone explain how to change the web interface listening address? Currently it is 127.0.0.1:5000 and there is no way to change this IP, I need to change the IP to my local one assigned to the VM (192.168.x.x) to connect to the cloudflare subdomain as a proxy through the tunnel. Possibly I should change all IP addresses from 127.0.0.1 to 192.168.x.x? It seems to me that only the one for the database should remain on 127.0.0.1. `Docker ps -a` shows that I do not have any ports assigned to postal containers, there is no mention in the documentation how to set them, and adding PORT and BIND_ADDRESS in config.yml does not work.

**Comment by willpower232** [comment]
You should be using something like caddy or nginx to handle web traffic and proxy it to port 5000.

Have you seen this part of the docs? https://docs.postalserver.io/getting-started/installation#caddy

**Comment by playaz44** [comment]
I saw this and I think I don't need Caddy when I use cloudflare as a proxy. Anyway, I can't access webui through Caddy either. I keep getting response timed out.

**Reply by willpower232** [reply]
I think you will be much better off using caddy with cloudflare, cloudflare definitely works better when you're proxying regular web traffic, it is possible that is causing some confusion currently.

If you use caddy (or whatever) to output your Postal UI then hopefully that should make add the extra ports you're expecting to see and also make it easier for you to enable cloudflare.

**Reply by playaz44** [reply]
Okay, I ran caddy according to the documentation and all I see is a white blank page. Nothing useful in logs.

 `postal logs web`:

```
web-1  | Loading config from /config/postal.yml
web-1  | Puma starting in single mode...
web-1  | * Puma version: 6.4.2 (ruby 3.2.2-p53) ("The Eagle of Durango")
web-1  | *  Min threads: 5
web-1  | *  Max threads: 5
web-1  | *  Environment: production
web-1  | *          PID: 8
web-1  | * Listening on http://127.0.0.1:5000
web-1  | Use Ctrl-C to stop
```

![image](https://github.com/postalserver/postal/assets/9553500/23f40e49-ad39-48fd-8dd4-4f6c1aaf7b0b)

**Reply by willpower232** [reply]
I presume you have forwarded caddy to postal and aren't using cloudflare at this point. It might be worth looking at the caddy logs to see what is or isn't happening.

**Comment by playaz44** [comment, marked as answer]
I was unable to gain access to the postal web-gui in any way. I switch to mailu, it works immediately without any problems.
